CodeGeneration/Sources/generate-swiftsyntax/templates/basicformat/BasicFormatFile.swift

```diff
@@ -137,6 +137,12 @@ let basicFormatFile = SourceFile {
     }
 
     FunctionDecl("open func requiresTrailingSpace(_ token: TokenSyntax) -> Bool") {
+      IfStmt("""
```

Comment:
Could you add a short comment that this only applies to empty dictionary literals? I was briefly confused because I thought that it also applied for dictionary elements.

Reply:
Done.